src/scripts/heroAnimation/__tests__/utils.test.js

```diff
@@ -126,10 +126,20 @@ describe("drawLines function", () => {
 });
 
 describe("getDistance function", () => {
-  it("should return distance between 2 points", () => {
+  it("should calculate distance between 2 points using hypot as default method", () => {
     const distance = utils.getDistance(point1, point2);
 
-    expect(distance).toEqual(200);
+    expect(distance).toEqual(141.4213562373095);
```

Comment:
If the mock data hasn't changed but this value has - does this mean you've changed the behaviour?

Reply:
@danbull that's right,  the behaviour changed it. Before it implemented this formula:

<img width="172" alt="screen shot 2018-05-14 at 15 41 42" src="https://user-images.githubusercontent.com/37550930/40004327-5792bf1e-578d-11e8-9bae-7428294f908a.png">

And I thought that we want to implement this [other for the refactor](https://en.wikipedia.org/wiki/Pythagorean_theorem#Euclidean_distance_in_various_coordinate_systems). 
<img width="280" alt="screen shot 2018-05-14 at 15 31 15" src="https://user-images.githubusercontent.com/37550930/40003698-e2d4b458-578b-11e8-87e1-af02f3ec211b.png">

Reply:
Ok :) still looks good to me!